### PR TITLE
README update

### LINKED ISSUE #53: Update README.md
The README.md should be updated to reflect the current state of the project:

- [ ] List tools used (KFP, KFTO, OCP, etc)
- [ ] List Prereqs
- [ ] Describe pipeline
- [ ] Explain how to deploy/use pipeline
- [ ] Add pipeline diagram

### PR DESCRIPTION
**Pull request by MichaelClifford**
This PR Updates the README to reflect the current state of the project. Closes  #53 #124 #195

### CODE REVIEW

**Review by sallyom — APPROVED** [review]
LGTM - we can update it from here if we missed anything!

File: README.md [hunk 1]
```diff
@@ -1,21 +1,43 @@
...
+* [Red Hat OpenShift AI](need_link)
+* [Red Hat OpenShift Serverless](need_link)
+* [Red Hat OpenShift Service Mesh](need_link)
+* [Red Hat - Authorino](need_link)
+* [NVIDIA GPU Operator](https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/getting-started.html)
+* [Node Feature Discovery](https://docs.openshift.com/container-platform/4.17/hardware_enablement/psap-node-feature-discovery-operator.html)
+
+#### Object Storage:
+
+Once the above operators have been successfully installed you will need to set up object storage for your models and pipeline artifacts. This solution requires object storage to be in place through [Noobaa](https://www.noobaa.io/).
```
Comment (sallyom) [inline README.md:27]: can be any S3 compatible storage, but we recommend nooba? and should we mention ODF?

File: README.md [hunk 2]
```diff
@@ -33,15 +55,150 @@ items:
...
+2. Deploy the Judge Model following these [instructions](/kubernetes_yaml/prometheus_serve/README.md).
+
+Once these two model servers are deployed, we need to add the following configmaps and secrets to our namespace so that the InstructLab pipeline can successfully communicate with each model.
+
+```yaml
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: teacher-server
+  namespace: <YOUR_NAMESPACE>
```
Comment (tumido) [inline README.md:77]: You don't need to specify the namespace every time.. if you submit a resource without `.metadata.namespace` it gets created in the currently selected namespace... so.. we can just omit it here and de-cluter.

File: README.md [hunk 3]
```diff
@@ -33,15 +55,150 @@ items:
...
+immutable: false
```
Comment (tumido) [inline README.md:78]: This doesn't need to be here, it's the default value.

File: README.md [hunk 4]
```diff
@@ -1,21 +1,43 @@
...
+
+#### Operators:
+The following Operators must be installed on your OpenShift cluster:
+
+* [Red Hat OpenShift AI](need_link)
+* [Red Hat OpenShift Serverless](need_link)
+* [Red Hat OpenShift Service Mesh](need_link)
+* [Red Hat - Authorino](need_link)
+* [NVIDIA GPU Operator](https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/getting-started.html)
+* [Node Feature Discovery](https://docs.openshift.com/container-platform/4.17/hardware_enablement/psap-node-feature-discovery-operator.html)
```
[the remark this answers is not in the document]
Comment (MichaelClifford) [reply README.md:23]: Sure! that is what I thought, but those other reqs were included in the earlier version. I'll remove for now.

**Comment by MichaelClifford** [comment]
@tumido @sallyom thanks for the feedback. Here is an updated version that is likely good enough for now that we can build on later if we need more info 😄

**Review by tumido — APPROVED** [review]
/lgtm
/approve

Love it. This is very nice and easy to follow. ❤️🙌

**Pull request merged**